src/en/getting-started-jaas.md

```diff
@@ -104,29 +90,63 @@ Press the blue button to deploy the changes.
 
 </table>
 
-Adjust your model name, if you desire, then choose a public cloud to deploy to.
+Adjust your model name, and choose a public cloud to deploy to.
 Enter your cloud credentials. Scroll down and click Deploy.
 
 ![juju_jaas_choose_cloud](./media/juju_jaas_choose_cloud.png)
 
-## Collaborate
 
-To share your model with another JAAS user you can share the standard Juju way,
-as described in [Users and models][users], or you can share directly from the
-JAAS GUI, like this.
+Your model is now being deployed for you. Now let's wire up the command line
+so that we can see this model and operate against it.
+
+
+## Accessing JAAS from the command line
+
+To use JAAS from the CLI requires that you install Juju on a local machine and
+connect that Juju install with your JAAS account.
```

Comment:
I think using "connect" instead of "register" is a bit confusing.

```
To use JAAS from the CLI requires that you install Juju on a local machine and
 register it with your JAAS account.
```